docs/schema/merging.md

```diff
@@ -28,9 +28,12 @@ In the release schema, `"omitWhenMerged": true` is declared on fields that must
 If `omitWhenMerged` is set to `false`, ignore it.
 
 ```{note}
```

Comment:
I think this whole admonition can be removed now as it simply repeats the instructions in the merge routine.